test/ambuda/views/proofing/test_talk.py

```diff
@@ -19,7 +19,7 @@ def test_thread__bad_project(client):
 
 
 def test_thread__bad_thread(client):
-    resp = client.get("/proofing/bad-project/talk/unknown")
+    resp = client.get("/proofing/test-project/talk/unknown")
```

Comment:
For consistency with the other tests in this file

```suggestion
    resp = client.get("/proofing/bad-project/talk/unknown")
```

Reply:
Here the piece being tested is that project ID is fine but the thread ID is bad